loader/db.go

```diff
@@ -117,12 +117,15 @@ func (conn *DBConn) executeSQL(ctx *tcontext.Context, queries []string, args ...
 				}
 				return true
 			}
-			ctx.L().Warn("execute statements", zap.Int("retry", retryTime),
-				zap.String("queries", utils.TruncateInterface(queries, -1)),
-				zap.String("arguments", utils.TruncateInterface(args, -1)),
-				log.ShortError(err))
-			tidbExecutionErrorCounter.WithLabelValues(conn.cfg.Name).Inc()
-			return retry.IsRetryableError(err)
+			if retry.IsRetryableError(err) {
+				ctx.L().Warn("execute statements", zap.Int("retry", retryTime),
+					zap.String("queries", utils.TruncateInterface(queries, -1)),
+					zap.String("arguments", utils.TruncateInterface(args, -1)),
+					log.ShortError(err))
+				tidbExecutionErrorCounter.WithLabelValues(conn.cfg.Name).Inc()
```

Comment:
only increase `tidbExecutionErrorCounter` for retryable error?

Reply:
fix it, it seems `tidbExecutionErrorCounter` should increase for every error [here](https://github.com/pingcap/dm/pull/266/files#diff-5f5bb90e4d87dad0cb25c377bb10679bL111),  and in `syncer/db.go` [`sqlRetriesTotal`](https://github.com/pingcap/dm/pull/266/files#diff-c3cc6fccd12f8e1679a575d1aaf8fd14R217-R225) only increase for retryable error